sdk/keyvault/keyvault-keys/package.json

```diff
@@ -28,6 +28,7 @@
     "LICENSE.txt",
     "README.md",
     "types/",
+    "types/keyvault-keys.d.ts",
```

Comment:
In theory, you shouldn't need the /types directory, just the .d.ts file.

Reply:
done!